maintainers/scripts/copy-tarballs.pl

```diff
@@ -189,6 +190,11 @@ sub uploadFile {
             next;
         }
 
+        if ($isPatch) {
+            print STDERR "skipping $url (support for patches is missing)\n";
```

Comment:
Basically a TODO?

Reply:
yes it's a TODO because it would be nice to have a copy on hydra I suppose